app/src/marketplace/components/ProductPage/ProductDetails/SocialIcons.jsx

```diff
@@ -0,0 +1,104 @@
+// @flow
+
+import React, { useMemo } from 'react'
+import styled from 'styled-components'
+
+import SvgIcon from '$shared/components/SvgIcon'
+import { type ContactDetails } from '$mp/flowtype/product-types'
+
+const Container = styled.div`
+    display: flex;
+    width: 100%;
+    height: 100%;
+    align-items: center;
+    justify-items: center;
+`
+
+const IconContainer = styled.div`
+    display: grid;
+    justify-items: center;
+    align-items: center;
+    grid-auto-columns: 24px;
+    grid-gap: 40px;
+    grid-auto-flow: column;
+`
+
+const Icon = styled(SvgIcon)`
+    width: 24px;
+    height: 24px;
+    color: #979797;
+`
+
+type Props = {
+    className?: string,
+    contactDetails: ContactDetails,
+}
+
+const getIconForUrl = (socialUrl: ?string) => {
+    if (socialUrl == null) {
+        return null
+    }
+
+    const url = socialUrl.trim().toLowerCase()
+
+    if (url.includes('github.com')) {
```

Comment:
Facebook is so old hat that it doesn't get its own icon? 😄

Reply:
Good point! Didn't even notice myself 😄 @mattatgit: Is there an icon for Facebook somewhere or do we want to boycott it totally? BTW, what should we do with links we don't have an icon for?

Reply:
@tumppi I guess we should have icons for all the major services, yeah. I guess our personal hatreds of companies like Facebook shouldn't prevent our users from being able to post links to those services. If you could make a ticket, we can make them for you.

Reply:
Here you go @mattatgit https://streamr.atlassian.net/browse/PLATFORM-1439. Thanks!